googleservices/cloudmonitoring/client.py

```diff
@@ -28,29 +28,33 @@ def write_timeseries_custon_instance_metric_double_value(self, project_id, metri
         unique_machine_id = get_gce_unique_id(http)
```

Comment:
Here I am starting the build the post data for this http request. The nicer way will be to you the client API (that I tried to update for this purpose), but it doesn't support it yet (the client api wraps the request so I will not have to). so I have to do it myself until they release a new version that has it.